dask_ml/model_selection/_search.py

```diff
@@ -1130,6 +1130,13 @@ def fit(self, X, y=None, groups=None, **fit_params):
             train/test set.
         **fit_params
             Parameters passed to the ``fit`` method of the estimator
+
+        Notes
```

Comment:
May need to remove if this is incorrect.

Reply:
I've revised this note to

> This class performs best when each cross validation split fits into RAM because fit is called on each cross validation split. For example, if using KFold with k chunks, a 1−1/k fraction of the dataset should fit into RAM.

Let me know if that's correct.